laptop-specific/battery.md

```diff
@@ -30,6 +32,22 @@ Once you've finally gotten your DSDT patched and battery working in macOS, it's
 
 ## Dual Battery
 
-Acidanthera has a guide on how to handle laptops with dual-batteries: [Link](https://github.com/acidanthera/VirtualSMC/blob/master/Docs/Dual%20Battery%20Support.md)
+Because macOS does not properly support systems with dual-batteries, you have to merge the two batteries in ACPI.
+
+Refer to this Acidanthera documentation for information on how to handle dual-battery laptops: [Link](https://github.com/acidanthera/VirtualSMC/blob/master/Docs/Dual%20Battery%20Support.md)
```

Comment:
`Refer to Acidanthera's documentation`
or
`Refer to the VirtualSMC documentation`
or something similar.

Reply:
Good suggestion. I agree because `Refer to the VirtualSMC documentation` is more applicable to the documentation in this case.